ivadomed/loader/loader.py

```diff
@@ -757,14 +756,20 @@ def __getitem__(self, index):
         # Update metadata_input with metadata_roi
         metadata_gt = imed_loader_utils.update_metadata(metadata_input, metadata_gt)
 
-        if self.task == "segmentation":
+
+
+
+        if self.task in ["segmentation", "pose_estimation"]:
             # Run transforms on images
             stack_gt, metadata_gt = self.transform(sample=seg_pair_slice["gt"],
                                                    metadata=metadata_gt,
                                                    data_type="gt")
             # Make sure stack_gt is binarized
             if stack_gt is not None and not self.soft_gt:
                 stack_gt = imed_postpro.threshold_predictions(stack_gt, thr=0.5).astype(np.uint8)
+            if self.task == "pose_estimation":
+                stack_gt = torch.moveaxis(torch.squeeze(stack_gt), -1, 0)
```

Comment:
Also, one other thing I came across while tracing in https://github.com/ivadomed/ivadomed/pull/787#discussion_r632740193: `ivadomed` has pinned [`torch==1.5.0`](https://github.com/ivadomed/ivadomed/blob/ccd82d22d31ffa53a2d6455ce3a94b97564d624a/requirements.txt#L15), but `moveaxis` was only added in late 2020 (https://github.com/pytorch/pytorch/pull/41480), so I don't think `1.5.0` has this. 

What version of `torch` do you have installed while developing?

Reply:
`moveaxis `is removed, not required anymore.